topology/zz-nobond/site_config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "site_name": "nz-kiwi",
+  "vlan_id": 100,
+  "tier1": {
+    "uplink_ports": [1],
+    "uplink_native": true,
+    #'upstream_lacp': 0,
```

Comment:
Are we allowed to comment in .json files? Can we get rid of the property if we are not using it?